public/index.html

```diff
@@ -0,0 +1 @@
+dummy
```

Comment:
why do u need this again?

Reply:
there's an api test that serves index.html

Reply:
added a spicy comment in there to explain